docs(review): review content (#1967)
scaleway/docs-content: dedibox-console/my-account/how-to/resolve-abuse.mdx

  paragraph: This page explains how to resolve Abuse tickets.
tags: resolve abuse complaint ticket dedibox-console 
dates:
  validation: 2023-07-24
  posted: 2020-03-12
categories:
  - account
---

Abuse tickets are complaints regarding the hosting of copyrighted content or engaging in illegal activities using Scaleway services.

<Message type="requirement">
  - You have an account and are logged into the [Dedibox console](https://console.online.net) 
</Message>

> 1. Click your username, located next to "logged in as" at the top right corner of the Dedibox console. Then, select **Abuses** from the pop-up menu that appears. A list of your abuse tickets displays.

Author: ```suggestion
1. Click your username, located next to "logged in as" at the top right corner of the Dedibox console. Then, select **Abuses** from the drop-down menu that displays. A list of your abuse tickets displays.
```

> 1. Click your username, located next to "logged in as," at the top right corner of the Dedibox console. Then, select **Abuses** from the pop-up menu that appears.

Author: ```suggestion
1. Click your username, located next to "logged in as" at the top right corner of the Dedibox console. Then, select **Abuses** from the pop-up menu that appears.
```

> 1. Click on your username, located next to "logged in as," at the top right corner of the Dedibox console. Then, select **Abuses** from the pop-up menu that appears.

Author: ```suggestion
1. Click your username, located next to "logged in as," at the top right corner of the Dedibox console. Then, select **Abuses** from the pop-up menu that appears.
```

> 2. A list of your abuse tickets displays. Click **Details** to view the specifics of the complaint.

Reviewer: This and step 4 do not start with a verb. I think it's best to send the first sentences back to the previous line.

> 1. Click your username, located next to "logged in as" at the top right corner of the Dedibox console. Then, select **Abuses** from the pop-up menu that appears.
> 2. A list of your abuse tickets displays. Click **Details** to view the specifics of the complaint.

Author: ```suggestion
1. Click your username, located next to "logged in as" at the top right corner of the Dedibox console. Then, select **Abuses** from the pop-up menu that appears. A list of your abuse tickets displays.
2. Click **Details** to view the specifics of the complaint.
```

3. Take note of the details provided and proceed with the necessary steps to address and resolve the issue. If you need guidance on resolving outgoing DDoS attacks from your server, refer to our documentation [how to prevent outgoing DDoS](/compute/instances/reference-content/preventing-outgoing-ddos/) for assistance.

> 4. After implementing the appropriate measures, respond to the author of the ticket with your actions taken and the resolution. To indicate that the ticket has been addressed, click **Resolve** to mark it as closed.

Author: ```suggestion
4. Describe the measures you have taken and provide an answer to the author of the ticket once the issue is solved. Click **Resolve** to mark the ticket as closed.
```

> 4. After implementing the appropriate measures, respond to the author of the ticket with your actions taken and the resolution. To indicate that the ticket has been addressed, click on Resolve to mark it as closed.

Author: ```suggestion
4. After implementing the appropriate measures, respond to the author of the ticket with your actions taken and the resolution. To indicate that the ticket has been addressed, click **Resolve** to mark it as closed.
```

    <Message type="note">

>       Refer to [this page](https://www.scaleway.com/en/abuse-notice/) for further information about Scaleway's abuse policy.

Author: ```suggestion
      Refer to [our abuse notice](https://www.scaleway.com/en/abuse-notice/) for further information about Scaleway's abuse policy.
```

>       For further information about Scaleway's abuse policy, you can refer to [this link](https://www.scaleway.com/en/abuse-notice/).

Reviewer 2: ```suggestion
      Refer to [this page](https://www.scaleway.com/en/abuse-notice/) for further information about Scaleway's abuse policy.
```

>       For further information about Scaleway's abuse policy, you can refer to [this link](https://www.scaleway.com/en/abuse-notice/).

Reviewer 3: ```suggestion
      For further information about Scaleway's abuse policy, you can refer to [our abuse notice](https://www.scaleway.com/en/abuse-notice/).
```
